docs/site-generator/Makefile

```diff
@@ -0,0 +1,155 @@
+# Default shell
```

Comment:
Which reminds me that we should try if this works with just `/bin/sh` instead of bash. If it does, we'll drop bash and use sh as it is already installed in the container and installing bash means ~200MB more in the container.